uiplib/utils/utils.py

```diff
@@ -77,3 +78,24 @@ def auto_flush(settings):
         delta = time.time() - os.path.getmtime(image_path)
         if delta > settings['days-to-autodel']:
             os.remove(image_path)
+
+
+def update_images(appObject):
```

Comment:
wel on second look all you need from appObject is settings, so could  just have setings in the param and send in setting instead of `appoObject`?